src/utilities/createInitialPermissions.js

```diff
@@ -296,6 +297,7 @@ const permissionsRoles = [
       'accessHgnSkillsDashboard',
       'manageFAQs',
       'manageHGNAccessSetup',
+      'interactWithSetFinalDayButton'
```

Comment:
```suggestion
      'setFinalDay'
```
I believe this was the key we used on the FE.